user/utils.py

```diff
@@ -1,11 +1,32 @@
 import json
+import jwt
+import bcrypt
 
 from django.http import JsonResponse
 
+from my_settings import SECRET_KEY, ALGORITHM
+from user.models import User
+
 def ValueErrorTypeChecking(data):
     if type(data['email']) != str:
         return JsonResponse({'MESSAGE' : 'INVALID_EMAIL_TYPE'}, status=400)
 
     if type(data['password']) != str:
         return JsonResponse({'MESSAGE' : 'INVALID_PASSWORD_TYPE'}, status=400)
 
+    if type(data['phone_number']) != str:
+        return JsonResponse({'MESSAGE' : 'INVALID_PHONE_NUMBER_TYPE'}, status=400)
+
+def SignInAuthorization(function):
+    def wrapper(self, request, *args, **kwargs):
+        try:
+            access_token = request.headers.get('Authorization')
+            payload      = jwt.decode(access_token, SECRET_KEY, algorithms=ALGORITHM)
+            signed_user  = User.objects.get(id=payload['id'])
```

Comment:
아래 코드에 대한 예외처리가 필요할 것 같습니다.
```suggestion
            signed_user  = User.objects.get(id=payload['id'])
```

Reply:
반영했습니다!